// python -m grpc_tools.protoc -I./ --python_out=. --grpc_python_out=. ./helloworld.proto

syntax="proto3";


// The greeting service definition.
service Greeter {
  // Sends a greeting
  rpc SayHello (HelloRequest) returns (HelloReply) {}
  // SayHelloAgain  
  rpc SayHelloAgain (HelloRequest) returns (HelloReply) {}
}

// The request message containing the user's name.
message HelloRequest {
  string name = 1;
}

// The response message containing the greetings
message HelloReply {
  string message = 1;
}


// 
service Model {
  rpc Inference (ModelRequest) returns (ModelReply) {}
} 

message ModelRequest {
  // string data = 1;
  bytes data = 1;
  float a = 2;
  float b = 3;
}

message ModelReply {
  // string result = 1;
  bytes data = 1;
  float c = 2;
}
